client/sites-dashboard/components/sites-table-row/docs/example.jsx

```diff
@@ -0,0 +1,156 @@
+import SitesTableRow from '../../sites-table-row';
+
+export default function SitesTableRowExample() {
+	const CONTAINER_STYLES = {
+		backgroundColor: 'white',
+		border: '1px solid lightgray',
+		padding: '16px',
+	};
+
+	const exampleSite = {
```

Comment:
@wojtekn , what do you think about moving these static constants outside the component?